ci/linux-sgx-centos8.2-gcc-release.jenkinsfile

```diff
@@ -40,8 +40,8 @@ node (node_label) {
             load '.ci/lib/config-release.jenkinsfile'
 
             load '../ci/stage-build-sgx.jenkinsfile'
-            load '../ci/stage-test.jenkinsfile'
             load '../ci/stage-test-sgx.jenkinsfile'
+            load '../ci/stage-test.jenkinsfile'
```

Comment:
what is this change for? generally unrelated refactoring should be done in a separate PR.